Multiple improvement

adrena-orex opened the pull request:
- Remove WSOL handling to simplify codebase
- Do a simulation before execute to optimize Computing Unit
- Add priority fee (max 1000 lamport) to boost up TX landing rate
- Calculate PnL and Liquidation Price for positions offchain
- Add a button to manually resolve staking rounds
- Fix loading for Locked Staking Threads in monitoring page
- Fix upgrading ALP staking

Adrena-Corto commented:
The button did not appear with the current condition for ALP not sure why, I just removed condition locally to trigger it

adrena-orex commented:
> The button did not appear with the current condition for ALP not sure why, I just removed condition locally to trigger it

Fixed, was a wrong trigger
